lib/trailer_vote/fixtures/application/vnd.trailervote.configuration.v4+json

```diff
@@ -0,0 +1,42 @@
+{
+  "configuration": {
+    "_links": {
+      "self": {
+        "href": "www.self_url.com"
+      },
+      "place": {
+        "href": "www.Place_url.com"
+      },
+      "push_manifest": {
+        "href": "www.push_manifest.com"
+      },
+      "products": {
+        "href": "www.push_manifest.com"
+      },
+      "product_lookup": {
+        "href": "www.product_lookup.com"
+      },
+      "persona": {
+        "href": "https://fake.trailervote.com:/api/personas/{uuid}/feedback",
+        "templated": true
+      },
+      "analytics": {
+        "href": "https://fake.trailervote.influxcloud.net:8086/analytics"
+      },
+      "telemetrics": {
+        "href": "https://fake.analytics.trailervote.com:8086/metrics"
+      },
+      "views_api": {
```

Comment:
Could we call this `persona_config`? The fact that it goes to the views service is an implementation detail.